services/util/util.go

```diff
@@ -250,7 +250,15 @@ func RunCommand(ctx context.Context, bin string, args []string, opts ...Option)
 	logger.Info("executing local command", "cmd", cmd.String())
 	run.Error = cmd.Run()
```

Comment:
I think we could also switch to using "Start()" and "Wait()", which return distinct errors for "couldn't execute the command" and "Command exited", but this is fine

Reply:
So...turns out that is equivalent (all Run() does is call Start()+Wait() effectively) so errors come out the same.

But...Thinking about it an ExitError isn't really an error in this API since we provide exit code and ExitError guarantees the stdout/stderr output is there if generated. So we shouldn't return an error for that case and nil it out.
It simplifies handling in exec/ansible but did add some work in process/fdbcli/packages.

PTAL